libs/liquidity/src/lib/liquidity-table.tsx

```diff
@@ -188,7 +188,9 @@ export const LiquidityTable = forwardRef<AgGridReact, LiquidityTableProps>(
       <AgGrid
         style={{ width: '100%', height: '100%' }}
         overlayNoRowsTemplate={t('No liquidity provisions')}
-        getRowId={({ data }) => data.id}
+        getRowId={({ data }: { data: LiquidityProvisionData }) =>
+          `${data.party.id}-${data.status}`
```

Comment:
_I think_ this might be a mistake? Are you deliberately changing this back? Madalina deployed today to fix the LP table which had an issue caused by this ID